extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -973,3 +885,114 @@ describes.sandboxed('amp-ad-network-doubleclick-impl', {}, () => {
     });
   });
 });
+
+
+describes.realWin('additional amp-ad-network-doubleclick-impl',
+    realWinConfigAmpAd, env => {
+      let doc;
+      let impl;
+      let element;
+      /**
+   * Creates an iframe promise, and instantiates element and impl, adding the
```

Comment:
Comments here and below seem misaligned. (This might predate this PR, but seems like a good time to fix, if you don't mind.)